app/controllers/calculator_constants_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CalculatorConstantsController < ApplicationController
+  def index
+    @calculator_constants = CalculatorConstant.all
+  end
+
+  def update
+    upload = Upload.where(csv_type: 'CalculatorConstant').first
```

Comment:
See https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/upload.rb#L72 for how to pull most recent Upload row

```suggestion
    upload = Upload.select('DISTINCT ON("csv_type") *').where({csv_type: 'CalculatorConstant', ok: true}).order(updated_at: :desc)
```

Reply:
Code's been removed!